com.playeveryware.eos/Documentation~/full_guide/01_starthere.md

```diff
@@ -0,0 +1,126 @@
+For more information on Player Groups, see [Epic's documentation on Player Groups](https://dev.epicgames.com/docs/dev-portal/product-management#player-groups).
+
+# Creating a Product
+
+By default your Organization will not contain any Products.
+While signed in to the `Epic Games Developer Portal`, and with your Organization selected, the `Create Product` button underneath the `Your products` header will start the process.
+After naming your Product, there will be a brief wait as Epic's back-end servers process the creation.
+When it's done, you can navigate to your Product to begin configuring it.
+
+## Additional Agreements Required
+
+At some point while navigating around your Product, there will be dialogues for Epic terms and services.
+There are several of these throughout the onboarding process, with specific domains.
+This will bring you to a Terms of Service page with detailed information about the agreement.
+In order to utilize many features of Epic Online Services, you will need to review and agree to these conditions.
+In order to use any features under `Epic Games Store`, there will be an agreement process that includes a possible submission fee.
+See [Epic's documentation on Get Started Overview](https://dev.epicgames.com/docs/epic-games-store/get-started/get-started-overview#onboarding-process) for more information.
+While this agreement is required to perform Epic Game Store operations, including publishing and selling your game, the EOS Plugin for Unity can be configured and utilized before this is agreed to.
+
+# Creating a Client Policy and Client
+
+The landing page for a Product will note your existing Product, Clients, Sandboxes, and Deployments.
+
+<img src="/com.playeveryware.eos/Documentation~/full_guide/epic_developerportal_productlanding.png" alt="The first thing in the Product landing page is your Product, Clients, Sandboxes, and Deployments. Other than Product, these likely haven't been configured yet."/>
+
+In order to get the most out of the EOS Plugin for Unity, you'll need to create several of these.
+Start by creating a Client by navigating to the `Clients` section.
+There you'll see your list of Clients and Client policies, which will be empty.
+
+For more information on Clients and Client Policies, see [Epic's Client Credential and Client Policy Management documentation](https://dev.epicgames.com/docs/dev-portal/client-credentials).
+Clients are essentially types of contexts for your game. For example, all users running your game from the store will be assigned a Client that represents their abilities.
+Each Client is assigned to one Client Policy.
+Client Policies are configurable sets of rules for what permissions a Client can perform. For example, a Client Policy determines the EOS SDK's ability to look up information about a player's friend list.
```

Comment:
"...about a player's friend list." -> "...about the friend list for a particular player." (helps remove contractions).

Reply:
Ermmm.... I'll implement as suggested, but "a player's friend list" is meant to be in regards to the player who launched your application. You can *also* use Client Policy to allow you to look up *other* player's friends list, but the primary need here was to identify the user of the application needs this permission for their own friends list to be used.